system/default_ad_api_helpers/src/routing_adaptor.cpp

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 Autoware Foundation
```

Comment:
Same here.

Reply:
The original code is part of [this file](https://github.com/autowarefoundation/autoware.universe/blob/main/planning/mission_planner/lib/mission_planner_base.cpp).

Reply:
https://github.com/autowarefoundation/autoware.universe/pull/1494#discussion_r940898682 This file is also mostly new. https://github.com/autowarefoundation/autoware.universe/pull/1494/commits/7eda408a79b0ed4e8f69d987c8cf413deeabd93c.